app/src/components/account/UserMenuAddress.vue

```diff
@@ -5,10 +5,7 @@
       :address="address || ''"
       :tooltip-text="addressType"
       :address-type="
-        network.network_type === 'polkadot' &&
-        session.addressRole &&
-        session.addressRole !== `stash/controller` &&
-        session.addressRole !== `none`
+        currentNetwork.network_type === 'polkadot' && session.addressRole
```

Comment:
```suggestion
        session.addressRole
```
if this is not null or undefined it should just work ... no?

Reply:
yeah, true. good observation